src/ts/components/subtitleoverlay.ts

```diff
@@ -33,9 +38,24 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     let subtitleManager = new ActiveSubtitleManager();
     this.subtitleManager = subtitleManager;
 
+    this.configureCea608Captions(player, uimanager);
+
     player.addEventHandler(player.EVENT.ON_CUE_ENTER, (event: SubtitleCueEvent) => {
       let labelToAdd = subtitleManager.cueEnter(event);
-
+      // If there is positionning data we assume we have CEA-608 style captions
+      labelToAdd.isCEA608 = event.position != null;
+
+      // In this block all the logic for CEA-608 captions is applied.
+      if (labelToAdd.isCEA608) {
+        let domElement = labelToAdd.getDomElement();
+        domElement.css({
+          'left': `${event.position.column * 0.6}em`,
+          // 6.66 = 100/15 the number of possible lines
+          'top': `${event.position.row * 6.66}%`,
+          'font-size': `${this.cea608fontSize}px`,
+        });
+        domElement.addClass(this.prefixCss(SubtitleOverlay.CLASS_CEA_608));
```

Comment:
Apply this class directly to the `SubtitleOverlay` instead of every `SubtitleLabel` (see comment above).